fastlane.gemspec

```diff
@@ -75,8 +75,8 @@ Gem::Specification.new do |spec|
   spec.add_dependency('colored') # colored terminal output
   spec.add_dependency('commander-fastlane', '>= 4.4.6', '< 5.0.0') # CLI parser
   spec.add_dependency('excon', '>= 0.45.0', '< 1.0.0') # Great HTTP Client
-  spec.add_dependency('faraday', '< 0.16.0') # Used for deploygate, hockey and testfairy actions
-  spec.add_dependency('faraday_middleware', '< 0.16.0') # same as faraday
+  spec.add_dependency('faraday', '~> 0.16.0') # Used for deploygate, hockey and testfairy actions
+  spec.add_dependency('faraday_middleware', '~> 0.16.0') # same as faraday
```

Comment:
Does this need the same version? Current seems 0.13.1: https://github.com/lostisland/faraday_middleware/releases

Reply:
I have an open PR with faraday_middleware to resolve that

Reply:
Ah, I see :) That should get rid of the last failure then.